src/components/Pages/Company.vue

```diff
@@ -0,0 +1,10 @@
+<script setup>
+import CompanyContent from '../Company/CompanyContent.vue';
+import CompanyJoin from '../Company/CompanyJoin.vue';
+</script>
+
+<template>
+    <CompanyContent />
+    <CompanyJoin />
+
```

Comment:
Are you using prettier for code formating?

Reply:
No, More of a vue language feature(volar) but not effective. I will fix it.